staging/src/k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/doc.go

```diff
@@ -17,4 +17,4 @@ limitations under the License.
 // Code generated by client-gen. DO NOT EDIT.
 
 // This package has the automatically generated clientset.
-package clientset
+package clientset // import "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
```

Comment:
these files are generated... unless the generator is updated as well, the scripts that verify the files match generated output will fail

Reply:
@liggitt ok, thank you.